common-arcana.lic

```diff
@@ -624,6 +616,19 @@ module DRCA
 
     cast_lifecycle_lambda.call('pre-prep', data, settings) if cast_lifecycle_lambda != nil
 
+    command = 'prep'
+    command = data['prep'] if data['prep']
+    command = data['prep_type'] if data['prep_type']     
+    
+    if data['prep'] == 'segue' || data['prep_type'] == 'segue'
```

Comment:
Why isn't this `if command == 'segue'` like on line 1878?

Reply:
Good catch! They should both be like 623.  Fixing.

Reply:
Should be!  Fixed!